src/test/java/org/apache/commons/text/ExtendedMessageFormatTest.java

```diff
@@ -373,14 +376,15 @@ private void checkBuiltInFormat(final String pattern, final Map<String, ?> regis
         } else {
             emf = new ExtendedMessageFormat(pattern, locale);
         }
-        assertEquals("format "    + buffer.toString(), mf.format(args), emf.format(args));
+        assertEquals("format " + buffer.toString(), mf.format(args), emf.format(args));
         assertEquals("toPattern " + buffer.toString(), mf.toPattern(), emf.toPattern());
     }
 
     /**
      * Replace MessageFormat(String, Locale) constructor (not available until JDK 1.4).
+     *
      * @param pattern string
-     * @param locale Locale
+     * @param locale  Locale
```

Comment:
Ditto.

Please consider reverting whitespace changes for _all_ `@param` lines where you only inserted whitespaces between the parameter name and its description.